test/integ/test_cursor.py

```diff
@@ -564,6 +563,42 @@ def test_variant(conn, db_parameters):
             cnx.cursor().execute("drop table {name}".format(name=name_variant))
 
 
+def test_geography(conn, db_parameters):
+    """Variant including JSON object."""
+    name_geo = db_parameters["name"] + "_geo"
+    with conn() as cnx:
+        cnx.cursor().execute(
+            f"""
+create table {name_geo} (geo geography)
+"""
```

Comment:
Remove the leading new line and also save the table cleanup necessary in our test code
```suggestion
            f"""\
create temporary table {name_geo} (geo geography)
"""
```